core/src/main/java/org/jivesoftware/spark/ui/TranscriptWindowEntry.java

```diff
@@ -38,13 +39,26 @@
      *
      * @param timestamp The timestamp of the entry (cannot be null).
      */
-    protected TranscriptWindowEntry( ZonedDateTime timestamp )
+    protected TranscriptWindowEntry( ZonedDateTime timestamp ){
+    	this(timestamp, false);    	
+    }
+    /**
+     * Constructs a new entry.
+     *
+     * When displayed in context of a conversation, chat entries are ordered by time. This is why the time component
+     * of an entry is non-optional. Note that multiple entries can exist that have the same time component.
+     *
+     * @param timestamp The timestamp of the entry (cannot be null).
+     * @param isDelayed Set true if timestamp come from delay extension.
```

Comment:
I'd change the description of the `isDelayed` argument, as it should be true for any message that is 'historic'. A 'delay' extension is just one way of expressing that. There might be other ways of expressing this too.